lasagne/layers/merge.py

```diff
@@ -91,7 +128,7 @@ def get_output_for(self, inputs, **kwargs):
             if coeff != 1:
                 input *= coeff
             if output is not None:
-                output += input
+                output = self.merge_function(output, input)
             else:
                 output = input
         return output
```

Comment:
> If you feel strongly about it then its fine :) There is no software engineer inside me.

I do :) I guess the list comprehension is easy to comprehend with the comment?

> I see your point with coupling.

The second problem with duplicating the code is that any changes in `ElemwiseMergeLayer` (whatever that could be) would need to be replicated in `ElemwiseSumLayer`, which may be forgotten. Again, that's the software engineer in me, `merge.py` is actually small and overseeable for practical matters. I'd still try to avoid code smell as much as possible.